app/service/rest.py

```diff
@@ -131,6 +132,19 @@ def get_platform_stats():
     return result
 
 
+@service_blueprint.route('/platform-stats-new', methods=['GET'])
+def get_new_platform_stats():
```

Comment:
I know the old route was in the service.rest but should we have a new route for platform admin stuff?